src/patternfly/components/FormControl/form-control.scss

```diff
@@ -243,6 +248,12 @@ $pf-c-form-control--m-clock--Coordinates: "M256 8C119 8 8 119 8 256s111 248 248
     }
   }
 
+  &[readonly].pf-m-plain {
```

Comment:
You can move this up as the last child under the existing `&[readonly]` and add `&.pf-m-plain`. It'll generate the same selector, and group these styles all under `readonly`.